lunar/distribution.yaml

```diff
@@ -675,5 +675,21 @@ repositories:
       url: https://github.com/ros/std_msgs.git
       version: groovy-devel
     status: maintained
+  urdfdom_py:
+    doc:
+      type: git
+      url: https://github.com/ros/urdf_parser_py.git
+      version: 0.3.2
```

Comment:
Got it about the malformed upstream URI; I'll fix that one over in the release repository.

As far as the documentation, there has been no change to the documentation, and no forks of the development, since Indigo.  So I think I'll change the version on all of them to "indigo-devel", which is the main development branch.  Patch coming.